### ISSUE TITLE
issue with fullbody parameters

### ISSUE THREAD

**Issue by mega** [MohammadWaleed/keycloak-admin-client#35]
Found an issue with fullbody parameters and created this patch which will make it work. 

Not sure this is the appropriate way to post a bug fix. 
sorry about that. 
thanks

```diff
 $ diff FullBodyLocation.php vendor/mohammad-waleed/keycloak-admin-client/src/Admin/Classes/FullBodyLocation.php
47c47
<         $this->jsonData[] = $this->prepareValue(
---
>         $this->jsonData = $this->prepareValue(
52c52,63
<         return $request->withBody(Psr7\stream_for(\GuzzleHttp\json_encode($this->jsonData)));
---
>         if ($this->jsonContentType && !$request->hasHeader('Content-Type')) {
>             $request = $request->withHeader('Content-Type', $this->jsonContentType);
>         }
>
>         return $request->withBody(Psr7\stream_for(\GuzzleHttp\json_encode($this->jsonData)));
>     }
>
>     public function after(CommandInterface $command, RequestInterface $request, Operation $operation)
>     {
>         return $request;
```

**Comment by MohammadWaleed** [comment]
Hello @mega  thsnk you for openning the issue, usually the right way of posting a bug fix would by by opening a new pull request describing the fix, anyway this is good.

could you please describe the issue you had and your fix ?

**Comment by mega** [comment]
Hi @MohammadWaleed 

Thanks for the follow up. This fix is related to those methods you have marked as ❌ which use a parameter for the body as json. Basically, when we used those methods in the request, the body would be empty because the `after` method that `FullBodyLocation` would inherit from `JsonLocation`.
Also, the jsonData object was being set as an array one time too many deep, ie, [[data]] instead of [data].

Hope this clarifies, if not I can try to explain it differently again.

Thank you again.

**Comment by MohammadWaleed** [comment]
@mega  That's great, thank you, i thought the issue was from keycloak so i didn't look into it much.

It'd be better if you opened a pull request with the fix, so you can be shown as a contributor in github

**Comment by MohammadWaleed** [comment]
fixed by #36
